README.md

```diff
@@ -38,20 +38,13 @@ All commands:
 
 1. Configure [direv](https://direnv.net/). Then copy `.sample.envrc` to a new `.envrc`. Don't forget to run `direnv allow` after any changes to `.envrc`.
 
-2. Install [docker](https://www.docker.com/get-started/) onto your machine, if you do not have it. Execute `docker-compose up -d` in the root of the repo. This starts the database. NOTE: If you've never run this command before, this will build a new image of the postgres database.
+2. Install [docker](https://www.docker.com/get-started/) onto your machine, if you do not have it. Execute `docker-compose up` in the root of the repo. This starts the database and pheonix servers.
 
-3. Install the tool versions necessary for this repo. If you are using [asdf](https://asdf-vm.com/), run these commands to install everything in the `.tool-versions` file:
-  * `asdf plugin add elixir`
-  * `asdf install`
+3. Create and migrate your database with the command: `docker compose run app mix ecto.migrate `.
 
-4. Start the Phoenix server:
+4. Download and install [ngrok](https://ngrok.com/download) onto your machine if you do not have it. Execute ngrok with the `PORT` that is configured as an enviornment variable: `ngrok http <PORT>`. This starts the ngrok reverse proxy so that way we can get the slack bot working over the web. Depending on how you've installed ngrok, you may have to navigate to the directory on your machine where `ngrok` resides and execute it with a prepended `./`.
 
-  * Install elixir dependencies then create and migrate your database with the command: `mix setup`. NOTE: You may be prompted to install `rebar3`. If so, install it.
-  * Start Phoenix endpoint with `mix phx.server`.
-
-5. Download and install [ngrok](https://ngrok.com/download) onto your machine if you do not have it. Execute ngrok with the `PORT` that is configured as an enviornment variable: `ngrok http <PORT>`. This starts the ngrok reverse proxy so that way we can get the slack bot working over the web. Depending on how you've installed ngrok, you may have to navigate to the directory on your machine where `ngrok` resides and execute it with a prepended `./`.
-
-6. Open up your [Slack app dev dashboard](https://api.slack.com). For `Captains Log Dev`, [this is the current one](https://api.slack.com/apps/A03L6Q2B6G1). If you do not know what this is, you should ask a teammate to add you to it or create a new slack app yourself. NOTE: reach out to the `#mojotime-release-notes` Slack channel to coordinate development since only one ngrok connection can be live at a time.
+5. Open up your [Slack app dev dashboard](https://api.slack.com). For `Captains Log Dev`, [this is the current one](https://api.slack.com/apps/A03L6Q2B6G1). If you do not know what this is, you should ask a teammate to add you to it or create a new slack app yourself. NOTE: reach out to the `#mojotime-release-notes` Slack channel to coordinate development since only one ngrok connection can be live at a time.
```

Comment:
I think we are missing a number 6 in this list of steps.

Reply:
Yes. One of the steps was removed and everything else got renumbered.